go/runtime/host/sgx/sgx.go

```diff
@@ -145,6 +146,23 @@ func (s *sgxProvisioner) loadEnclaveBinaries(rtCfg host.Config) ([]byte, []byte,
 	return sgxs, sig, nil
 }
 
+func (s *sgxProvisioner) discoverSGXDevice() (string, error) {
+	// Different versions of Intel SGX drivers provide different names for
+	// the SGX device.  Autodetect which one actually exists.
+	sgxDevices := []string{"/dev/sgx", "/dev/sgx/enclave", "/dev/isgx"}
```

Comment:
And maybe also `/dev/sgx_enclave` (see https://github.com/fortanix/rust-sgx/blob/master/intel-sgx/sgxs-loaders/src/isgx/mod.rs#L444).